src/main/java/chess/controller/ChessController.java

```diff
@@ -22,33 +23,20 @@ public void run() {
     private void play(ChessGame chessGame) {
         try {
             List<String> commands = InputView.inputGameFunction();
-            Function function = Function.from(commands.get(0));
-
-            doFunction(chessGame, function, commands);
+            Function function = Function.from(commands.get(COMMAND_INDEX));
+            execute(chessGame, function, commands);
         } catch (IllegalArgumentException | IllegalStateException e) {
             System.out.println(e.getMessage());
         }
     }
 
-    private void doFunction(ChessGame chessGame, Function function, List<String> commands) {
-        if (function == Function.START) {
-            chessGame.start();
-            OutputView.printBoard(chessGame.getValue());
-        }
-
-        if (function == Function.MOVE) {
-            chessGame.move(Coordinate.of(commands.get(1)), Coordinate.of(commands.get(2)));
-            OutputView.printBoard(chessGame.getValue());
-        }
-
-        if (function == Function.STATUS) {
-            StatusCalculator statusCalculator = new StatusCalculator(chessGame.getValue());
-            OutputView.printStatus(statusCalculator.createStatus());
-        }
-
-        if (function == Function.END) {
-            chessGame.end();
+    private void execute(ChessGame chessGame, Function function, List<String> commands) {
+        if (!function.isStatus()) {
```

Comment:
```suggestion
        if (function.isNotStatus()) {
```
(저번 피드백에 의하면) 이런 형태의 제공이 더 잘 읽힐거라 생각해요 :)